src/main/kotlin/geojsonkt/BBox.kt

```diff
@@ -47,3 +55,42 @@ fun BBox.toPolygon(): Polygon {
     return Polygon(arrayOf(arrayOf(northWest, southWest, southEast, northEast, northWest)))
 }
 
+// Added bbox things
+
+fun GeoJson.bbox() = {
+    when(this) {
+        is Point -> throw UnsupportedOperationException("Can not calculate BBox of Point Geometry type: ${this::class.java.name}")
+        is LineString -> bbox(coords(this))
+        is Polygon -> bbox(coords(this))
+        is MultiPoint -> throw UnsupportedOperationException("Can not calculate BBox of MultiPoint Geometry type: ${this::class.java.name}")
```

Comment:
Add support for bbox on MultiPoint. Even if turf.js doesn't support it, it seems easy enough to implement.